src/engine/source/base/parseEvent.hpp

```diff
@@ -8,20 +8,18 @@
 
 namespace base::parseEvent
 {
-
-constexpr char EVENT_AGENT_ID[] {"/agent/id"};
-constexpr char EVENT_AGENT_NAME[] {"/agent/name"};
-constexpr char EVENT_LOG[] {"/event/original"};
 constexpr char EVENT_QUEUE_ID[] {"/wazuh/queue"};
-constexpr char EVENT_REGISTERED_IP[] {"/agent/registeredIP"};
-constexpr char EVENT_ORIGIN[] {"/wazuh/origin"};
+constexpr char EVENT_LOCATION_ID[] {"/wazuh/location"};
```

Comment:
I think this should be `full_location` and not location, (as we have the log and full_log), since it includes information of the agent and not only of the module.

Reply:
It is called as described in the documentation
https://documentation.wazuh.com/current/development/message-format.html